docs/cloud-native-security/cloud-native-security-index.asciidoc

```diff
@@ -1,7 +1,38 @@
 [[cloud-native-security-overview]]
-
 = Cloud native security
-Elastic’s cloud security capabilities help you to improve your cloud security posture by comparing your cloud configuration to best practices, and allow you to monitor and investigate your Linux deployments inside and outside of Kubernetes.
+Elastic’s cloud security solutions help you to improve your cloud security posture by comparing your cloud configuration to best practices, and scanning for vulnerabilities. They also help you monitor and investigate your cloud workloads inside and outside of Kubernetes.
+
+This page describes what each solution does and how to implement it.
+
+[discrete]
+== Cloud Security Posture Management (CSPM)
+Cloud Security Posture Management helps you to harden your cloud security posture by assessing the configuration of your cloud accounts against security best practices defined by the Center for Internet Security (CIS). Once enabled, it runs regular scans which generate data that can inform your remediation efforts.
+
+To learn more, refer to <<cspm, CSPM>>.
+
+[discrete]
+== Kubernetes Security Posture Management (KSPM)
+Kubernetes Security Posture Management helps you to harden the security posture of your Kubernetes deployments by assessing their configuration against security best practices defined by the Center for Internet Security (CIS). Once enabled it runs regular scans which generate data that can inform your remediation efforts.
```

Comment:
```suggestion
Kubernetes Security Posture Management helps you to harden the security posture of your Kubernetes deployments by assessing their configuration against security best practices defined by CIS. Once enabled, it runs regular scans which generate data that can inform your remediation efforts.
```

Reply:
I know we already defined CIS earlier on the page, but I think someone might skip the first section so it wouldn't hurt to define it here as well